Problems/Programmers/67256/ans_67256_BNR.PY

```diff
@@ -0,0 +1,47 @@
+'''
+https://programmers.co.kr/learn/courses/30/lessons/67256
+
+'''
+def solution(numbers, hand):
+    answer = ''
+    key = [[1,2,3],[4,5,6],[7,8,9],['*',0,'#']]
```

Comment:
```suggestion
    key = [[1,4,7,'*'],[2,5,8,0],[3,6,9,'#']]
```
List 구성을 세로 우선으로 했다면, 16번줄에서의 탐색 구문(in~)을 제거해서 복잡도를 낮출 수 있을 듯.

Reply:
세로로 하는 게 루프하나 뺄 수 있겠네요!